docs/documentation/documentation_guidelines.md

```diff
@@ -0,0 +1,214 @@
+
+3. Click on "compare against forks" at the top, and change the head (on the right) to *their* repository.
+
+4. Now you can see the commits that *they* made since you forked and started working, if any; from this page, start a pull request, which is essentially you, *asking you*, if you want their commits added to your fork.
+
+You can see this process explained in further detail [here](https://github.com/KirstieJane/STEMMRoleModels/wiki/Syncing-your-fork-to-the-original-repository-via-the-browser) and [here](https://stackoverflow.com/questions/20984802/how-can-i-keep-my-fork-in-sync-without-adding-a-separate-remote/21131381#21131381).
+
+# For Professionals
+
+This section describes how to accomplish the above workflow on your local machine, and test it there before committing changes.
+
+## Getting Started
+
+One preferred way to add documentation to PennLINC is to spin up and run a webpage on your local machine, which will dynamically and automatically update as you write.
+You can then test and review changes as they would appear on the website without risk of damaging the public-facing site and without going back and forth with the commit workflow above.
+Sounds great, huh?
+
+To do this, though, there is a little bit of overhead, mainly:
+
+1. A web browser (duh)
+2. A program called `ruby`
+3. A program called `jekyll`
+
+`ruby` is a popular language for web development and has a multitude of templates available, so it makes sense that this website is built on one of them.
+`jekyll` on the other hand is what's called a *static site generator*.
+It takes written documents and parses them into static websites (i.e. ones that don't need interactive features or a database backend).
+`jekyll` uses `ruby` to create pretty websites, while Github uses `jekyll` to publish websites on the internet for users and projects.
+Make sense?
+If not, it's no matter.
+You will only need to do a few main things to get started with local testing for contributing to this site.
+
+## How To Test Locally
+
+First, clone this repository to your local machine:
+
+```
+git clone https://github.com/PennLINC/PennLINC.github.io.git
+```
+
+Next, make sure you have `Ruby` installed; the best way to check this is to do `which ruby`,
+and if there is no `ruby` program found,
+it's recommended you install it with `brew` (most Linux and MacOS machines already use `brew` as a package manager akin to `pip` or `npm`,
+see [this guide](https://brew.sh/) to find out how to install `brew` first if you don't have it).
+
+To install ruby, just do:
+
+```
+brew install ruby
+
+# YOU SHOULD PROBABLY ALSO ADD IT TO YOUR PATH LIKE SO:
+
+echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> ~/.bash_profile
+```
+
+Now install `jekyll` using a ruby "gem", by doing the following:
+
+```
+# PERMISSIONS MAY BE REQUIRED
+
+gem install --user-install bundler jekyll
```

Comment:
This didn't work for me - lots of version/ dependency issues

Reply:
Yeah I didn't use these instructions either. The changes here are just from me moving the files.